src/Transport/Config/DefaultConfigurationValues.cs

```diff
@@ -34,7 +33,8 @@ void ApplyDefaultValuesForSerialization(SettingsHolder settings)
 
         void ApplyDefaultValuesForAddressing(SettingsHolder settings)
         {
-            settings.SetDefault(WellKnownConfigurationKeys.Topology.Addressing.Partitioning.Namespaces, new List<string>());
+            settings.SetDefault(WellKnownConfigurationKeys.Topology.Addressing.Partitioning.Namespaces, new NamespacesDefinition());
+            settings.SetDefault(WellKnownConfigurationKeys.Topology.Addressing.UseLogicalNamespaceName, (Func<NamespaceDefinition, string>)(x => x.ConnectionString));
```

Comment:
check how ASQ is doing it, you can apply the same logic imo

- incoming: https://github.com/Particular/NServiceBus.AzureStorageQueues/pull/42/files#diff-2527dbd34187f07376b335d39883a92fR151
- outgoing: https://github.com/Particular/NServiceBus.AzureStorageQueues/pull/42/files#diff-582a0cef3422849f67816681c602843dR114